Ginger/GingerCore/ALM/JIRA/Bll/JiraExportManager.cs

```diff
@@ -157,22 +162,25 @@ public ComparisionOperator GetComparisionOperator(string compareOperator)
         /// <param name="defectId">string defectId</param>
         void AttachScreenshotsToDefects(string screenshots, string defectId)
         {
-            string[] screenshotsPaths = screenshots.Split(',');
-            foreach (string screenshotsPath in screenshotsPaths)
-            {
-                if (screenshotsPath.Length > 0)
-                {
-                    var attachments = jiraRepObj.AddAttachment(ALMCore.DefaultAlmConfig.ALMUserName, ALMCore.DefaultAlmConfig.ALMPassword, ALMCore.DefaultAlmConfig.ALMServerURL, defectId, screenshotsPath);
-                    if (attachments.DataResult != null)
-                    {
-                        Reporter.ToLog(eLogLevel.DEBUG, "Added Attachment to defect");
-                    }
-                    else
-                    {
-                        Reporter.ToLog(eLogLevel.ERROR, "Failed to add attachment to defect -" + attachments.AuthenticationResponseObj.ErrorDesc);
-                    }
-                }
-            }
+            // Olego - 9/23/20 - temporary marking
+            // to check with Arvind
+
+            //string[] screenshotsPaths = screenshots.Split(',');
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6280018)